packages/extension-ui/src/Popup/Accounts/AccountsTree.tsx

```diff
@@ -65,9 +65,6 @@ export default styled(AccountsTree)`
     flex-direction: column;
     
   .accountWichCheckbox {
-    display: flex;
-    align-items: center;
```

Comment:
This I think is outdated styles, left after some refactor - the `accountWichCheckbox` div only ever one child. I'd even remove the redundant div, but unfortunately it was used as an _onclick_ target (instead of using that lone child) and changing that would require more testing.